modules/pipeline/providers/edgepipeline_register/interface.go

```diff
@@ -111,15 +111,15 @@ func (p *provider) GetEdgeBundleByClusterName(clusterName string) (*bundle.Bundl
 
 func (p *provider) GetAccessToken(req apistructs.OAuth2TokenGetRequest) (*apistructs.OAuth2Token, error) {
 	return &apistructs.OAuth2Token{
-		AccessToken: p.EdgeTaskAccessToken(),
+		AccessToken: p.ClusterAccessKey(),
```

Comment:
split EdgeTaskAccessToken to a new file.
or delete it if useless.

Reply:
splited